src/System.Private.CoreLib/src/System.Private.CoreLib.csproj

```diff
@@ -196,6 +196,11 @@
     <Compile Include="System\Reflection\TypeInfo.cs" />
     <Compile Include="System\Reflection\TypeInfo.Workaround.cs" />
     <Compile Include="System\Reflection\Runtime\CustomAttributes\RuntimeImplementedCustomAttributeData.cs" />
+    <Compile Include="System\Resources\MissingManifestResourceException.cs" Condition="'$(IsProjectNLibrary)' != 'true'" />
```

Comment:
If possible, I'd prefer to avoid forking CoreRT/ProjectN here. I suspect having these in ProjectN won't hurt anything (but please confirm)